samples/apps/copilot-chat-app/webapp/README.md

```diff
@@ -5,6 +5,73 @@ production deployments.
 
 See [../README.md](../README.md) for complete instructions on setting up and running the application.
 
+## How to use HTTPS for local development
+If you want to run Copilot Chat with HTTPS, you need to create a certificate and sign it with a Certificate Authority (CA) that is trusted locally by your device and browser. You can do that easily with [mkcert](https://github.com/FiloSottile/mkcert#installation).
+
+1. From an elevated shell, run
+
+    ### Windows
+    On Windows, use [Chocolatey](https://chocolatey.org/)
+    ```
+    choco install mkcert
+    ```
+
+    ### MacOS
+    On macOS, use [Homebrew](https://brew.sh/)
+    ```
+    brew install mkcert
+    ```
+
+    For installation on other OS and advanced topics (i.e., supported root stores), see the [official mkcert installation guide](https://github.com/FiloSottile/mkcert#installation). 
+
+1. Create a new local CA by running
```

Comment:
[nit] certificates are usually new for most people. I would recommend using as few acronyms as possible.
```suggestion
1. Create a new local certificate authority by running
```